### PR TITLE
Fix matleroux issues

### PR DESCRIPTION
**Pull request by krokrob**
@ssaunier this is a so "club-internet" bug, please review!

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -42,6 +42,12 @@
 #  twitter_nickname       :string
 #  noindex                :boolean          default(FALSE), not null
 #  private_bio            :text
+#  status                 :string
+#  mood                   :text
+#  linkedin_nickname      :string
+#  facebook_nickname      :string
+#  pre_wagon_experiences  :jsonb            is an Array
+#  post_wagon_experiences :jsonb            is an Array
 #
```
Comment (krokrob) [inline app/models/user.rb]: Maybe I should delete this?

File: db/schema.rb [hunk 2]
```diff
@@ -308,6 +308,12 @@
     t.string   "twitter_nickname"
     t.boolean  "noindex",                default: false, null: false
     t.text     "private_bio"
+    t.string   "status"
+    t.text     "mood"
+    t.string   "linkedin_nickname"
+    t.string   "facebook_nickname"
+    t.jsonb    "pre_wagon_experiences",                               array: true
+    t.jsonb    "post_wagon_experiences",                              array: true
   end
```
Comment (krokrob) [inline db/schema.rb]: And this?

File: app/models/user.rb (hunk 1 above)
Comment (ssaunier) [inline app/models/user.rb]: yes!

File: app/policies/ordered_list_policy.rb [hunk 3]
```diff
@@ -12,6 +12,6 @@ def update?
   private
 
   def user_is_manager_or_admin?
-    user.admin || user.cities.map(&:slug).include?(record.name.match(/([a-z]+)/)[1])
+    user.admin || user.cities.map(&:slug).include?(record.name.match(/([a-z-]+)/)[1])
```
Comment (ssaunier) [inline app/policies/ordered_list_policy.rb]: Fuck, sorry...

File: db/schema.rb (hunk 2 above)
Comment (ssaunier) [inline db/schema.rb]: Yes!

**Pull request merged**